src/composables/useIsDarkTheme/index.ts

```diff
@@ -54,10 +54,10 @@ const useInternalIsDarkTheme = createSharedComposable(() => useIsDarkThemeElemen
  */
 export function useIsDarkTheme(): DeepReadonly<Ref<boolean>> {
 	const isDarkTheme = useInternalIsDarkTheme()
-	const enforcedTheme = inject(INJECTION_KEY_THEME)
+	const enforcedTheme = inject(INJECTION_KEY_THEME, computed<''>(() => ''))
```

Comment:
Why not `undefined`, to not create additional computed?
```suggestion
	const enforcedTheme = inject(INJECTION_KEY_THEME, undefined)
```

Reply:
much better!